packages/@okta/vuepress-site/docs/reference/api/roles/index.md

```diff
@@ -3122,9 +3254,12 @@ The following are the supported resources.
 |                         | All customizations                <br><ApiLifecycle access="ea" />  | `orn:${partition}:idp:${yourOrgId}:customizations`                                    |                                                                                                                                                         |
 | Workflows               | All Delegated Flows               <br><ApiLifecycle access="ea" />  | `orn:${partition}:workflow:${yourOrgId}:flows`                                       |                                                                                                                                                         |
 |                         | A specific Delegated Flow         <br><ApiLifecycle access="ea" />  | `orn:${partition}:workflow:${yourOrgId}:flows:${flowId}`                             |                                                                                                                                                         |
+| Governance              | All Certifications                <br><ApiLifecycle access="ea" />  | `orn:$partition$:governance:$orgId$:certifications`                                   |                                                                                                                                                         |
```

Comment:
Same question here as to whether these should be "Access Certifications" and "Access Requests" to match the product or whether they should match the ORN names